plugin/errors.go

```diff
@@ -0,0 +1,55 @@
+package plugin
+
+import "fmt"
+
+type errNotFound string
+
+func (name errNotFound) Error() string { return fmt.Sprintf("plugin %q not found", string(name)) }
```

Comment:
Put the function body on its own line for consistency with the rest of the file.